README.md

```diff
@@ -69,21 +69,31 @@ _Pre-requisite: OpenShift 4.10 or above_
 
 If the OpenShift Console is detected in the cluster, a console plugin is deployed when a `FlowCollector` is installed. It adds new pages and tabs to the console:
 
-- A flow table, with powerful filtering and display options
+#### Overview dashboard
 
-![Flow table](./docs/assets/network-traffic-main.png)
+This dashboard shows overall, aggregated statistics on the cluster traffic. The stats can be refined with comprehensive filtering and display options. Different levels of aggregations are available: per node, per namespace, per owner or per pod/service). It allows to identify biggest talkers in different contexts: for instance, top X inter-namespace flows, or top X pod-to-pod flows within a namespace, etc.
```

Comment:
`It allows to identify biggest` this is just one of the available functionalities --- either describe all the available panels  or state that this is an example